.github/workflows/unit_tests.yml

```diff
@@ -204,7 +204,7 @@ jobs:
     - name: Install MNE (previous stable)
       if: matrix.mne-version == 'mne-prev-stable'
       run: |
-        git clone --single-branch --branch maint/1.5 https://github.com/mne-tools/mne-python.git
+        git clone --single-branch --branch maint/1.6 https://github.com/mne-tools/mne-python.git
```

Comment:
why did we opt to not get this from PyPI?

Reply:
we need the full repo, including stuff that's not shipped in the dist